examples/Sarcastic-bot/Ballerina.toml

```
@@ -0,0 +1,14 @@
+[package]
+org = "kanishka"
```

We usually use the `wso2` as the orgname for the examples
```suggestion
org = "wso2"
```